controllers/api_v1/login_test.go

```diff
@@ -1,66 +1,39 @@
 package api_v1_test
 
 import (
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net/http"
 	"net/url"
-	"os"
 	"testing"
 
 	"github.com/gutakk/go-google-scraper/config"
-	errorconf "github.com/gutakk/go-google-scraper/config/error"
-	"github.com/gutakk/go-google-scraper/controllers"
-	"github.com/gutakk/go-google-scraper/controllers/api_v1"
 	"github.com/gutakk/go-google-scraper/db"
-	"github.com/gutakk/go-google-scraper/helpers/log"
 	"github.com/gutakk/go-google-scraper/models"
-	"github.com/gutakk/go-google-scraper/oauth"
 	testConfig "github.com/gutakk/go-google-scraper/tests/config"
 	testDB "github.com/gutakk/go-google-scraper/tests/db"
+	"github.com/gutakk/go-google-scraper/tests/fabricator"
 	testHttp "github.com/gutakk/go-google-scraper/tests/http"
+	testjson "github.com/gutakk/go-google-scraper/tests/json"
```

Comment:
To be consistent, the casing should be:

```suggestion
	testJson "github.com/gutakk/go-google-scraper/tests/json"
```

Reply:
Updated testJson in 47c1b0be9ed95166672e085e19c38521a3fffc21